infra/charts/turing-init/values.yaml

```diff
@@ -7,7 +7,7 @@ image:
   # -- Docker registry for Turing cluster init
   registry: ghcr.io/
   # -- Docker image repository for Turing cluster init
-  repository: gojek/turing-cluster-init
+  repository: gojek/turing/cluster-init
```

Comment:
oops! thanks

Reply:
Actually if we were to do this, can we delete the old images, so that it's not confusing to the user on which image to use.

Reply:
There are no old images, the CI pipeline publishes it under `gojek/turing/cluster-init`, it was just that helm chart contains wrong values for it.